src/System.ServiceModel.Primitives/ref/System.ServiceModel.Primitives.cs

```diff
@@ -199,7 +199,7 @@ public abstract partial class SecurityToken
         public abstract DateTime ValidFrom { get; }
         public abstract DateTime ValidTo { get; }
         //public virtual bool CanCreateKeyIdentifierClause<T>() where T : System.IdentityModel.Tokens.SecurityKeyIdentifierClause { return default(bool); }
-        //public virtual T CreateKeyIdentifierClause<T>() where T : System.IdentityModel.Tokens.SecurityKeyIdentifierClause { return default(T); }
```

Comment:
@mconnew I either just missed adding this method or there was some reason for not adding it, but I didn't see any notes and I can't specifically remember if there was something discussed. Any reason you can think of for not having added this?